src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -340,6 +379,70 @@ public void validate(FailureCollector collector) {
       }
     }
 
+    public void validateContentType(FailureCollector failureCollector) {
+      switch (format) {
+        case FORMAT_AVRO:
+          if (!contentType.equalsIgnoreCase(CONTENT_TYPE_APPLICATION_AVRO)) {
+            failureCollector.addFailure(String.format("Valid content type for avro is %s",
```

Comment:
Please add period to the end of all the validation error messages

Reply:
PR Updated.